apis/core/v1alpha1/condition.go

```diff
@@ -71,6 +71,7 @@ type Condition struct {
 
 	// A Message containing details about this condition's last transition from
 	// one status to another, if any.
+	// +optional
```

Comment:
is this `+optional` marker only for doc generation, or is it used in any other context?

Reply:
I believe it's used by a handful of static analysis tools, though I don't know which ones. It's "highly recommended" by the API conventions per https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required